### PR TITLE
Add govuk front end to repo

### PR DESCRIPTION
**Pull request by Joanneyoung01**
- Adds govuk front end to the repo

Guidance has been left in at this point as it may prove useful, particularly when creating the new form pages as this has a template available

### CODE REVIEW

File: src/views/base.njk [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  <link rel="stylesheet" href="/stylesheets/govuk-frontend-4.7.0.min.css" />
+{% endblock %}
+
+{% from "govuk/components/header/macro.njk" import govukHeader %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/skip-link/macro.njk" import govukSkipLink %}
+{% from "govuk/components/header/macro.njk" import govukHeader %}
+{% from "govuk/components/footer/macro.njk" import govukFooter %}
+
+{% block skipLink %}
```
Comment (di-aholme) [inline src/views/base.njk:11]: Would be a good addition to have a body tag wrapping the content for semantic html, lines 11 - 60

Comment (Joanneyoung01) [reply src/views/base.njk:11]: have added in, thank you!

File: src/app.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+const express = require("express");
+let nunjucks = require("nunjucks");
+const path = require("path");
+const { configureNunjucks } = require("./config/nunjucks");
+const app = express();
+let router = express.Router();
```
[the remark this answers is not in the document]
Comment (Joanneyoung01) [reply src/app.js]: Removed router now that we've deleted this out for now and renamed to const

File: package.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
+{
+  "scripts": {
+    "dev": "node src/app.js",
+    "serve": "node serve",
+    "start": "node src/app.js nodemon server.js -e html,js,njk,css",
```
[the remark this answers is not in the document]
Comment (Joanneyoung01) [reply package.json]: applied, thanks!

File: src/app.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (Joanneyoung01) [reply src/app.js]: applied, thanks!

File: src/data/session-data-defaults.js [hunk 4]
```diff
@@ -0,0 +1,5 @@
+module.exports = {
```
[the remark this answers is not in the document]
Comment (Joanneyoung01) [reply src/data/session-data-defaults.js]: applied, thanks!

File: src/filters.js [hunk 5]
```diff
@@ -0,0 +1,10 @@
+//
```
[the remark this answers is not in the document]
Comment (Joanneyoung01) [reply src/filters.js]: applied, thanks!

File: src/routes.js [hunk 6]
```diff
@@ -0,0 +1,9 @@
+//
```
[the remark this answers is not in the document]
Comment (Joanneyoung01) [reply src/routes.js]: applied, thanks!

File: src/views/base.njk [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+}) }}
+{% endblock %}
+
+
+{% block main %}
+  <div class="govuk-width-container">
+    <main id="main-content" class="govuk-main-wrapper">
+      <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+          <h1 class="govuk-heading-xl">Homepage</h1>
```
[the remark this answers is not in the document]
Comment (Joanneyoung01) [reply src/views/base.njk]: Good catch! Moved to home

File: src/views/layouts/main.html [hunk 8]
```diff
@@ -0,0 +1,3 @@
+{# For guidance on how to use layouts see:
```
[the remark this answers is not in the document]
Comment (Joanneyoung01) [reply src/views/layouts/main.html]: deleted :) thanks

**Review by di-fabs — APPROVED** [review]

File: src/views/base.njk (hunk 1 above)
Comment (di-aholme) [reply src/views/base.njk:9]: Body tag present in DOM

**Review by di-aholme — APPROVED** [review]

**Pull request merged**